### PR TITLE
Project 2 rest of mobile

### PR DESCRIPTION
**Pull request by Cerdtruir**
Built the first part of my portfolio project according to the spec from portfolio/2_mobile.md

live link: https://htmlpreview.github.io/?https://github.com/Cerdtruir/My-Portfolio/blob/Project-2-rest-of-mobile/index.html

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -87,12 +95,80 @@ li {
...
+  background-color: #fff;
+  border-color: #396df2;
+  color: #396df2;
+  padding: 12px;
+  font-weight: 500;
+  font-size: 17px;
+  width: 130px;
+  height: 50px;
+  border-radius: 16px;
+  border-width: 1px;
```
Comment (ikraamg) [inline style.css:162]: The border looks a bit thin. You can optionally increase it.

**Review by ikraamg — APPROVED** [review]
Your portfolio site is coming together nicely 👌 

The only changes required are to the borders of the language skills and each portfolio card. They both need a border radius on them.

image

image

File: style.css (hunk 1 above)
Comment (Cerdtruir) [reply style.css:162]: > Your portfolio site is coming together nicely ok_hand
> 
> The only changes required are to the borders of the language skills and each portfolio card. They both need a border radius on them.
> 
> image
> 
> image

Im already 2 branches ahead, and fixed this in the current branch im on. Do you you think i should fix it on this branch for the experience?

Comment (Cerdtruir) [reply style.css:162]: the border according to the spec is supposed to be 1px

Comment (Cerdtruir) [reply style.css:162]: > 

nevermind, i fixed it here

Comment (ikraamg) [reply style.css:162]: It's a good idea to get some practice as you did 👍 . 
Often, when you are working with other team members on the same feature or section of code, you need to rebase newer branches on top of each other.

**Pull request merged**